x-pack/plugins/actions/server/lib/audit_events.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AuditEvent, EventOutcome, EventCategory, EventType } from '../../../security/server';
+
+export enum ConnectorAction {
+  CREATE = 'connector_create',
+  GET = 'connector_get',
+  UPDATE = 'connector_update',
+  DELETE = 'connector_delete',
+  FIND = 'connector_find',
+  EXECUTE = 'connector_execute',
+}
+
+type VerbsTuple = [string, string, string];
+
+const eventVerbs: Record<ConnectorAction, VerbsTuple> = {
+  connector_create: ['create', 'creating', 'created'],
```

Comment:
Is there a reason we're using snake case here?
I know we have some places where we use snake case and some where we use camel case, but I know Core have been working to standerdise these things, so thought I'd ask 🤷

Reply:
Agree this is unfortunate but ECS standards uses snake case and this action name will be included in the log output so was more important for me to follow ECS standard than Kibana internal casing.